R/functions-analyses.R

```diff
@@ -141,8 +133,8 @@ fitJags  <-  function(lmModelMatrix, data, run=TRUE, outputFileName='mod.bug') {
 ###############
 # PAPER NUMBERS
 ###############
-extractNumbersList  <-  function(metRates, outputPath) {
-    load(outputPath)
+extractNumbersList  <-  function(metRates, output) {
```

Comment:
Instead of reading in data using ` load(outputPath)`, you can just pass it in directly. Remake removes the need to save intermediate objects to file